### PR TITLE
Move to certbot instead of concert

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: docs/generate-lets-encypt-certificate-using-certbot-for-minio.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- Install Certbot from [here](https://certbot.eff.org/)
+
+## 2. Dependencies
+- Port 443 for https needs to be open and available at time of executing `certbot`.
+- Certbot needs root access while executing because only root is allowed to bind to any port below 1024.
+- We will be using our own domain ``myminio.com``  as an example in this recipe. Replace with your own domain under your setup.
+
+## 3. Recipe Steps
+
+### Step 1: Install certbot as shown below.
```
Comment (NitishT) [inline docs/generate-lets-encypt-certificate-using-certbot-for-minio.md]: Title can be kept as just `Install Certbot`

File: docs/generate-lets-encypt-certificate-using-certbot-for-minio.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+total 4
+lrwxrwxrwx 1 root root  37 Aug  2 09:58 cert.pem -> ../../archive/myminio.com/cert4.pem
+lrwxrwxrwx 1 root root  38 Aug  2 09:58 chain.pem -> ../../archive/myminio.com/chain4.pem
+lrwxrwxrwx 1 root root  42 Aug  2 09:58 fullchain.pem -> ../../archive/myminio.com/fullchain4.pem
+lrwxrwxrwx 1 root root  40 Aug  2 09:58 privkey.pem -> ../../archive/myminio.com/privkey4.pem
+-rw-r--r-- 1 root root 543 May 10 22:07 README
+```
+
+### Step 4: Set up SSL on Minio Server with the certificates.
+The generated keys via Certbot needs to be placed inside users home directory at ``${HOME}/.minio/certs``
```
Comment (NitishT) [inline docs/generate-lets-encypt-certificate-using-certbot-for-minio.md]: The certificate and key generated via Certbot needs to be placed inside user's home directory

File: docs/generate-lets-encypt-certificate-using-certbot-for-minio.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## 2. Dependencies
+- Port 443 for https needs to be open and available at time of executing `certbot`.
+- Certbot needs root access while executing because only root is allowed to bind to any port below 1024.
+- We will be using our own domain ``myminio.com``  as an example in this recipe. Replace with your own domain under your setup.
+
+## 3. Recipe Steps
+
+### Step 1: Install certbot as shown below.
+Install certbot by following the documentation at https://certbot.eff.org/
```
Comment (NitishT) [inline docs/generate-lets-encypt-certificate-using-certbot-for-minio.md]: `Certbot` instead of `certbot`

**Comment by harshavardhana** [comment]
Addressed the requested changes @NitishT

**Pull request merged**